### DISCUSSION TITLE
4.3.4/4.3.5 model runs available for comparison?

### DISCUSSION THREAD

**Discussion by jansteinhauser** [magpiemodel/magpie#334] [Q&A] [answered]
Hello again!

I'd like to ensure that, despite some warnings here and there, my MAgPIE is running and working as intended. For v4.0, there were model run outputs and validation files available alongside the release paper. Is something similar available for v4.3.4 and v4.3.5?
I'd like to recreate some of your runs and compare the outputs. So, I guess, I'd need any changed config and script files and, of course, the outputs. Is that possible?

As always, thanks a lot!
Jan

**Comment by tscheypidi** [comment, marked as answer]
Hi Jan,

to my knowledge we only have these runs for v4.0 publicly availably, but I agree with you that it would be useful to also have it for the other versions available.

Best, Jan
